api/pkg/s3/router.go

```diff
@@ -46,4 +46,7 @@ func RegisterRouter(ws *restful.WebService) {
 	ws.Route(ws.DELETE("/{bucketName}/?lifecycle").To(handler.RouteBucketDelete)).Doc("Delete lifecycle configuration from the bucket")
 
 	ws.Route(ws.PUT("/{bucketName}/?versioning").To(handler.RouteBucketPut)).Doc("Create Versioning configuration for the bucket")
+
+	// router for SSE
+	ws.Route(ws.PUT("/{bucketName}/?DefaultEncryption").To(handler.RouteBucketPut)).Doc("Set default encryption on bucket")
```

Comment:
the url path should comply with the specification 'https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketEncryption.html'

Reply:
hmm...i have a query here
```
why do we want to make all our APIs and header parameters etc. the same as what AWS currently supports ? specifically,
why only AWS ? why not HWS or GCP or Azure ?
i agree we need to functionally match-up, yes. but that can be done with different APIs and parameters
One more thing, say we do match AWS today, what is our mechanism to always keep it matching, to always be in sync with AWS ? What if AWS changes their API 2 weeks later, we keep playing catch-up ?
```

Reply:
IMHO, currently, aws s3 is the de facto standard, there are many client tools or SDK which works based on s3 standard, if we do not compatible with s3 standard, that means those tools cannot work for us. So currently, we'd better be compatible with it, for the future, it depends.

Reply:
tracked in #751